services/api/package.json

```diff
@@ -27,16 +27,18 @@
     "@koa/router": "^9.3.1",
     "@sentry/node": "^6.2.1",
     "bcrypt": "^5.0.0",
+    "bytes": "^3.1.0",
+    "chalk": "^4.1.0",
+    "dateformat": "^4.5.1",
```

Comment:
same for this date format is just for local development, google logging deals with formatting

Reply:
this is just for dev... could instrumentation expose this? or is it something cloud logging does internally?